fuzz/README.md

```diff
@@ -61,6 +61,7 @@ To run fuzzing using `cargo-fuzz / libFuzzer`, run
 
 ```shell
 rustup install nightly # Note: libFuzzer requires a nightly version of rust.
+export RUSTFLAGS="--cfg=fuzzing --cfg=secp256k1_fuzz --cfg=hashes_fuzz"
```

Comment:
I ran into this as well, but it seems I don't need to add `--cfg=fuzzing`. Does it work for you without that?